reconcile/test/test_typed_queries/test_status_board.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from reconcile.gql_definitions.status_board.status_board import (
+    AppV1,
+    EnvironmentV1,
+    NamespaceV1,
+    ProductV1,
+    StatusBoardProductV1,
+    StatusBoardProductV1_StatusBoardAppSelectorV1,
+)
+from reconcile.typed_queries.status_board import get_selected_app_names
+
+
+@pytest.fixture
+def status_board_product():
+    return StatusBoardProductV1(
```

Comment:
nit: We have a factory method available to all test functions for generated classes that could help here https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/test/conftest.py#L117

Reply:
That looks great, refactored the code